src/main/scala/de/uhh/lt/conll/CoNLLParser.scala

```diff
@@ -0,0 +1,69 @@
+package de.uhh.lt.conll
```

Comment:
pretty much the same thing applies to conll.
it is by intention programmed in a way that it can be used outside of josimtext. to indicate this it is outside of the jst package. again this will absolutely change in the future, but for now, it would communicate this property better, if it is not included in jst. but again it seems to be a bit random whether to include or not. do you still want me to move in it into jst?